libraries/botbuilder-core/tests/activityHandlerBase.test.js

```diff
@@ -44,52 +39,55 @@ describe('ActivityHandlerBase', function() {
         onUnrecognizedActivity = false;
     });
 
-    it('should throw an error if context is not passed in', done => {
+    it('should throw an error if context is not passed in', async function () {
         const bot = new ActivityHandlerBase();
 
-        bot.run().catch(error => {
-            if (error.message !== 'Missing TurnContext parameter') {
-                done(error);
-            } else {
-                done();
-            }
-        });
+        try {
+            await bot.run();
```

Comment:
I realize I'm moving the goal posts on you, so feel free to push this out to a separate issue or pull request. That said, this test can/should be written using `assert.rejects`.

Reply:
That's not a problem at all. I guess it depends on how much test cleanup we want to do. I don't mind doing it and I think some occasional housekeeping is good. I see 270 results in 61 files for some use of `catch` in tests that would likely be better with `rejects()`

Reply:
Separate issue and PR then - it'll make it easier/quicker to review I think.

Reply:
Tracking issue: https://github.com/microsoft/botbuilder-js/issues/3426